.bazelrc

```diff
@@ -10,3 +10,4 @@ build --per_file_copt='external*@-w'
 # This workaround is needed due to https://github.com/bazelbuild/bazel/issues/4341
 build --per_file_copt="external/com_github_grpc_grpc/.*@-DGRPC_BAZEL_BUILD"
 build --http_timeout_scaling=5.0
+build --per_file_copt=".*@-g"
```

Comment:
we can remove this option now; otherwise the size of the generated binary will be extremely huge

Reply:
You are right. I will delete it.